### PR TITLE
Added prometheus ports to omnirpc helm chart

### PR DESCRIPTION
**Pull request by bsgrigorov**

### CODE REVIEW

File: charts/omnirpc/templates/deployment.yaml [hunk 1]
```diff
@@ -53,9 +53,10 @@ spec:
             {{- toYaml .Values.env | nindent 12 }}
           {{- end }}
           ports:
-            - name: http
-              containerPort: 5080
-              protocol: TCP
+          {{- range .Values.service.ports }}
```
Comment (trajan0x) [inline charts/omnirpc/templates/deployment.yaml:56]: do we want to export the metrics at the service level, or the container level? Not sure which we're doing here but container is definitely preferable

File: charts/omnirpc/templates/servicemonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" }}
```
Comment (trajan0x) [inline charts/omnirpc/templates/servicemonitor.yaml:1]: think we may want to allow users do disable?

**Review by trajan0x — CHANGES_REQUESTED** [review]

File: charts/omnirpc/templates/deployment.yaml (hunk 1 above)
Comment (bsgrigorov) [reply charts/omnirpc/templates/deployment.yaml:56]: service level is more standard pod level should be used only if we don't have a service

File: charts/omnirpc/templates/servicemonitor.yaml (hunk 2 above)
Comment (bsgrigorov) [reply charts/omnirpc/templates/servicemonitor.yaml:1]: ok

File: charts/omnirpc/templates/deployment.yaml (hunk 1 above)
Comment (trajan0x) [reply charts/omnirpc/templates/deployment.yaml:56]: I don't think so. The container label exists for a reason by default on prom. This way you can do aggegates across all your containers.

If you do service level you're just hitting a random container every scrape interval and you miss any outliers in your histogram

**Review by trajan0x — CHANGES_REQUESTED** [review]

**Review by trajan0x — APPROVED** [review]

**Review by trajan0x — APPROVED** [review]

**Pull request merged**